ext/hathach/tusb_port.cpp.in

```diff
@@ -12,6 +12,7 @@
 
 #include <modm/platform/device.hpp>
 #include <modm/architecture/interface/interrupt.hpp>
+#include <modm/architecture/interface/atomic_lock.hpp>
```

Comment:
Need to add `:architecture:atomic` to [the module dependencies](https://github.com/modm-io/modm/blob/develop/ext/hathach/module.lb#L59).